lib/core/src/debug/monitor.rs

```diff
@@ -591,13 +614,56 @@ impl Sampler for WasmMemory {
     fn value    (&self) -> f64         { self.value }
     fn check    (&self) -> ValueCheck  { self.value_check }
     fn min_size (&self) -> Option<f64> { Some(100.0) }
-    fn end      (&mut self, _time:f64)  {
+    fn end      (&mut self, _time:f64) {
         let memory: Memory      = wasm_bindgen::memory().dyn_into().unwrap();
         let buffer: ArrayBuffer = memory.buffer().dyn_into().unwrap();
         self.value              = (buffer.byte_length() as f64) / (1024.0 * 1024.0);
-        self.value_check        =
-            if      self.value <=  50.0 { ValueCheck::Correct }
-            else if self.value <= 100.0 { ValueCheck::Warning }
-            else                        { ValueCheck::Error   };
+        self.value_check        = self.check_by_treshold(50.0,100.0);
```

Comment:
wait, those are threshold values - warning and error. How would you like the code to look like here?

Reply:
1. Then, the `check_by_thershold` parameters should be "warn_threshold" "error_threshold" instead of t1, t2,
2. Still its goot to have such constants somehow separated. It's easier to find what value we are supposed to change when we will need to change these thresholds. And the fps limits are repeated. So maybe make const FPS_WARN_THRESHOLD, FPS_ERROR_THRESHOLD, and if you want, similar for memory.

Reply:
makes perfect sense, thanks